_blog/2018/soft-multitenancy.md

```diff
@@ -0,0 +1,298 @@
+---
+title: "Istio Multitenancy Support"
+overview: Using kubernetes namespace and RBAC to create an Istio soft multitenancy environment
+publish_date: March 21, 2018
+subtitle: Using multiple Istio control planes and RBAC to create multitenancy
+attribution: John Joyce and Rich Curran
+
+order: 92
+
+layout: blog
+type: markdown
+redirect_from: "/blog/soft-multitenancy.html"
+---
+{% include home.html %}
+Multitenancy is commonly used in many environments across many different applications,
+but the implementation details and functionality provided on a per tenant basis does not
+follow one model in all environments.  The Kubernetes multitenancy [working group](
+https://github.com/kubernetes/community/blob/master/wg-multitenancy/README.md)
+is working to define the multitenant use cases and functionality that should be available
+within Kubernetes. From their work so far it is clear that only a "soft multitenancy" is
+possible due to the inability to fully protect against malicious containers or workloads
+gaining access to the other pods or kernel resources.
+
+## Defining Soft Multitenancy for Istio
+While discussing the strengths and benefits of deploying applications on top of Istio it
+became apparent that there are multitenant use cases for Istio even within this "soft"
+environmental that does not provide absolute protection amongst tenants.  A viable use case
+for this scenario is shared corporate infrastructure where malicious actions are not
+expected but a clean separation of the tenants is still required. A couple different
+multitenant models could be considered.
+1.	A single mesh with multiple applications, one for each tenant on the mesh. The cluster
```

Comment:
Istio doesn't really define the notion of application at this point. In the context of k8s, would this basically equal a Deployment?